lib/index-cartesian.js

```diff
@@ -1,19 +1,29 @@
 'use strict';
 
-var Plotly = require('./core');
+var core = require('./core');
 
-Plotly.register([
+core.register([
+    // traces
     require('./bar'),
     require('./box'),
     require('./heatmap'),
     require('./histogram'),
     require('./histogram2d'),
     require('./histogram2dcontour'),
-    require('./image'),
-    require('./pie'),
     require('./contour'),
     require('./scatterternary'),
-    require('./violin')
+    require('./violin'),
+    require('./image'),
+    require('./pie'),
+
```

Comment:
As you can see, `image` and `pie` used to be registered in a different order in `casrtesian` bundles.